frontend/js/src/settings/brainzplayer/BrainzPlayerSettings.tsx

```diff
@@ -0,0 +1,216 @@
+        you have one available).
+      </p>
+      <p>
+        <Switch
+          id="enable-spotify"
+          disabled={!SpotifyPlayer.hasPermissions(spotifyAuth)}
+          value="spotify"
+          checked={spotifyEnabled}
+          onChange={(e) => setSpotifyEnabled(!spotifyEnabled)}
+          switchLabel={
+            <span
+              className={`text-brand ${spotifyEnabled ? "text-success" : ""}`}
+            >
+              <FontAwesomeIcon icon={faSpotify} />
+              &nbsp;Spotify
+            </span>
+          }
+        />
+        <br />
+        <small>
+          Spotify requires a premium account to play full songs on third-party
+          websites. To sign in, please go to the{" "}
+          <Link to="/settings/music-services/">music services page</Link>.
+        </small>
+      </p>
+      {/* <p>
+        <Switch
+          id="enable-apple-music"
+          value="apple-music"
+          disabled={!AppleMusicPlayer.hasPermissions(appleAuth)}
+          checked={appleMusicEnabled}
+          onChange={(e) => setAppleMusicEnabled(!appleMusicEnabled)}
+          switchLabel={
+            <span className={`text-brand ${enabled ? "text-success":""}`}>
+              <FontAwesomeIcon icon={faApple} />
+              &nbsp;Apple Music Player
+            </span>
+          }
+        />
+        <br/>
+        <small>
+          Apple Music requires a premium account to play full songs on
+          third-party websites. To sign in, please go to the{" "}
+          <Link to="/settings/music-services/">music services page</Link>. You
+          will need to sign in every 6 months.
+        </small>
+      </p> */}
+      <p>
+        <Switch
+          id="enable-soundcloud"
+          value="soundcloud"
+          disabled={!SoundcloudPlayer.hasPermissions(soundcloudAuth)}
+          checked={soundcloudEnabled}
+          onChange={(e) => setSoundcloudEnabled(!soundcloudEnabled)}
+          switchLabel={
+            <span
+              className={`text-brand ${
+                soundcloudEnabled ? "text-success" : ""
+              }`}
+            >
+              <FontAwesomeIcon icon={faSoundcloud} />
+              &nbsp;Soundcloud
+            </span>
+          }
+        />
+        <br />
+        <small>
+          Soundcloud requires a free account to play full songs on third-party
+          websites. To sign in, please go to the{" "}
+          <Link to="/settings/music-services/">music services page</Link>
+        </small>
+      </p>
+      <p>
+        <Switch
+          id="enable-youtube"
+          value="youtube"
+          checked={youtubeEnabled}
+          onChange={(e) => setYoutubeEnabled(!youtubeEnabled)}
+          switchLabel={
+            <span
+              className={`text-brand ${youtubeEnabled ? "text-success" : ""}`}
+            >
+              <FontAwesomeIcon icon={faYoutube} />
+              &nbsp;Youtube
+            </span>
```

Comment:
Does changing ther logo color violate the branding guidelines?

Reply:
Hm, good call, I would guess probably,